### DISCUSSION TITLE
How to adjust 'ignore while typing' sensitivity of trackpad?

### DISCUSSION THREAD

**Discussion by prwood** [elementary/settings-mouse-touchpad#233] [Q&A]
I have elementary OS running on a 2011 MacBook Pro. Mostly things are working well, but one issue I'm having is that the trackpad is picking up stray taps from my palm while I'm typing. I went to Settings -> Hardware -> Mouse and Touchpad -> Touchpad -> Ignore: While typing and checked it off. It does make things a bit better but it's still picking up a lot of palm taps while typing.

Is there anywhere I can tweak how this 'ignore while typing' feature works so that it is less likely to pick up taps while typing?

**Comment by danirabbit** [comment]
If you turn off "Ignore while typing" the trackpad will be active and receive input while you are typing. If you want taps to be ignore while you are typing, you need to turn that feature back on
